### ISSUE TITLE
mason.ini for boost includes -L flag

### ISSUE THREAD

**Issue by jfirebaugh** [mapbox/mason#296]
mason.ini for boost 1.62.0 includes `ldflags=-L{prefix}/lib`. However this is a header-only package and no `lib` directory exists, so this flag generates a linker warning:

```
ld: warning: directory not found for option '-L/Users/john/Development/mapbox-gl-native/mason_packages/headers/boost/1.62.0/lib'
```

cc @springmeyer

**Comment by springmeyer** [comment]
Working on a fix.

**Comment by springmeyer** [comment]
-> #297

**Comment by springmeyer** [comment]
Solved by #297
